example/ios/example.xcodeproj/project.pbxproj

```diff
@@ -285,13 +288,19 @@
 		83CBB9F71A601CBA00E9B192 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 0940;
+				LastUpgradeCheck = 1130;
 				ORGANIZATIONNAME = Facebook;
 				TargetAttributes = {
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = UL428N5782;
+						ProvisioningStyle = Automatic;
 						TestTargetID = 13B07F861A680F5B00A75B9A;
 					};
+					13B07F861A680F5B00A75B9A = {
+						DevelopmentTeam = UL428N5782;
```

Comment:
Could we remove the DevelopmentTeam option so anyone can use the example project? 😃

Reply:
Well i though you always need to use a Development Team?

Reply:
indeed, development team is required to run the example with a real device. however, other people won't be able to use your development team id

Reply:
Odd, Xcode always forces me to select one?...But this should matter know... the developer has to adjust the xcode-project anyway :)